packages/nimble-components/src/utilities/models/converter.ts

```diff
@@ -0,0 +1,17 @@
+import { booleanConverter, ValueConverter } from '@microsoft/fast-element';
+
+export const optionalBooleanConverter: ValueConverter = {
+    toView(value: unknown): string {
+        return typeof value === 'boolean' ? value.toString() : '';
+    },
+
+    fromView(value: string): boolean | null | undefined {
+        if (value === undefined) {
+            return undefined;
+        }
+        if (value === null) {
+            return null;
+        }
```

Comment:
We are trying to advocate that it is unexpected behavior that FAST lets null values through, lets not propagate that behavior

```suggestion
        if (value === undefined || value === null) {
            return undefined;
        }
```